test/core/run_tests.py

```diff
@@ -79,16 +79,20 @@ def run_cmd(mode):
         with open('check_flow.py', 'w') as f:
             f.write(formatter.check_code)
         with open('.coveragerc', 'w') as f:
-            f.write("[run]\ndisable_warnings = module-not-measured\n")
+            f.write("[run]\ndisable_warnings = module-not-measured")
```

Comment:
nit: a reason for removing "\n" at the end?

Reply:
fixed (i had more changes on this line originally but then rolled them back)